src/amuse_simplex/packages/amuse-simplex/pyproject.toml

```diff
@@ -21,6 +21,9 @@ distance = "{next_version}.dev{distance}+{vcs}{rev}"
 dirty = "{base_version}+d{build_date:%Y%m%d}"
 distance-dirty = "{next_version}.dev{distance}+{vcs}{rev}.d{build_date:%Y%m%d}"
 
+[tool.versioningit.vcs]
+method = { module = "version_helper", value = "get_amuse_version", module-dir = "amuse_simplex/support/shared" }
+
```

Comment:
was this fix only needed specifically for Simplex?

Reply:
Yeah, I converted the rest using a script so they were all good, then fixed Simplex (which had a bunch of issues) using the template I'd made previously for converting the codes, which didn't have these extra lines, so they were missing only there. I did check the rest to make sure :smile:.